tests/memex/__init___test.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+import mock
+
+import memex
+
+
+def test_set_groupfinder(pyramid_config):
```

Comment:
That's fair enough. I think that concern is reason enough to make the default context something like this:

```
class DefaultGroupContext(object):
    def __init__(self, name):
        self.name = name

    def __acl__(self):
        if self.name == '__world__':
            return [(security.Allow, security.Authenticated, 'write'),
                    (security.Allow, security.Everyone, 'read'),
                    security.DENY_ALL]
        return [security.DENY_ALL]